### PR TITLE
🎨 Reword laminr docs

### PR DESCRIPTION
**Pull request by Zethson**
- Adds a pre-commit config to run Prettier
- Rewords the introduction in the README

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1 @@
...
-.Rdata
-.httr-oauth
-.quarto
-*.Rproj
-
-experiments
-docs
-/doc/
-/Meta/
+*.html
```
Comment (rcannood) [inline .gitignore:1]: Revert these changes :bow:

```suggestion
.Rproj.user
.Rhistory
.Rdata
.httr-oauth
.quarto
*.Rproj

experiments
docs
/doc/
/Meta/
*.html
```

**Review by rcannood — CHANGES_REQUESTED** [review]
Almost everything looks good to me! Only I would undo the changes to the .gitignore, as most of them are relevant for R package development and pkgdown rendering.

File: .gitignore (hunk 1 above)
Comment (Zethson) [reply .gitignore:1]: I don't even know how that happened - sorry! Of course.

**Review by rcannood — APPROVED** [review]
LGTM!
